django_multitenant/tests/urls.py

```diff
@@ -1,5 +1,12 @@
 # -*- coding: utf-8 -*-
-from django.conf.urls import include, url
+import django
+from django.conf.urls import include
+
+if django.VERSION >= (3, 1):
```

Comment:
above you check `django.VERSION > (3,2)` which seems semantically the same.
Could we standardize on 1 variant of testing this?

I think I like the variant used at this instance better than above, as it tells me slightly more precise what is included.

How does this work with patch versions in `3.1`? (assuming semver)

Reply:
> How does this work with patch versions in 3.1? (assuming semver)

They are tuple compares, which work like this:
```
>>> (3, 1, 0) >= (3, 1)
True
>>> (3, 1, 1) >= (3, 1)
True
>>> (3, 0, 5) >= (3, 1)
False
```